### ISSUE TITLE
Edit user account fields are empty

### ISSUE THREAD

**Issue by byeight** [nilsteampassnet/TeamPass#1958]
### Steps to reproduce
1. While logged in as admin, click Manage Users
2. Click on an existing user: Edit User account

### Expected behaviour
User can be edited

### Actual behaviour
The fields "Login, Name, Last name, Email, Roles"  are empty.
![2017-10-06_15-17-11](https://user-images.githubusercontent.com/29922616/31279520-9d408e2e-aaa9-11e7-88dc-4d608d0a892e.png)

### Server configuration
**Operating system**:
Windows Server 2012 R2
**Web server:**
Apache 2.4.27
**Database:**
MySQL 5.7.19
**PHP version:**
PHP 7.0.23
**Teampass version:** 
2.1.27.9

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this message the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by leoalvarezg** [comment]
I have exactly the same behavior on a fresh install with a totally different server.

`Ubuntu 16.04 4.4.0-96-generic

Server version: Apache/2.4.18 (Ubuntu)
Server built:   2017-09-18T15:09:02

PHP 7.0.22-0ubuntu0.16.04.1 (cli) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
 with Zend OPcache v7.0.22-0ubuntu0.16.04.1, Copyright (c) 1999-2017, by Zend Technologies`

**Comment by nilsteampassnet** [comment]
This is a bug present in development branch (future 2.1.27.9).
It has been fixed in commit 8be3cc3d48bb62a8b95225d4410526cae9d5138d done the 3 Oct
